packages/ddp-client/livedata_common.js

```diff
@@ -5,4 +5,5 @@ LivedataTest.SUPPORTED_DDP_VERSIONS = DDPCommon.SUPPORTED_DDP_VERSIONS;
 // This is private but it's used in a few places. accounts-base uses
 // it to get the current user. Meteor.setTimeout and friends clear
 // it. We can probably find a better way to factor this.
-DDP._CurrentInvocation = new Meteor.EnvironmentVariable;
+DDP._CurrentMethodInvocation = DDP._CurrentInvocation = new Meteor.EnvironmentVariable;
```

Comment:
I would split this into two lines. One creating `DDP._CurrentMethodInvocation` and the other setting `DDP._CurrentInvocation` and commenting that this for backwards compatibility since X version because it was used as a public API by many packages.

We should maybe also change everywhere else in the core code to use `DDP._CurrentMethodInvocation`. But maybe somebody else from MDG could comment on this first.